hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -3036,6 +3039,8 @@ func reconcilecontrolPlaneOperatorIngressOperatorRoleBinding(binding *rbacv1.Rol
 func reconcileCAPICluster(cluster *capiv1.Cluster, hcluster *hyperv1.HostedCluster, hcp *hyperv1.HostedControlPlane, infraCR client.Object) error {
 	// We only create this resource once and then let CAPI own it
 	if !cluster.CreationTimestamp.IsZero() {
+		// make sure cluster is not paused.
+		cluster.Spec.Paused = false
```

Comment:
this would only unpause if the deletion timestamp is non-zero, I think we want the opposite

Reply:
this is checking the creationTimestamp, or are you refering to something else?

Reply:
sorry I immediately thought it was the deletionTimestamp